fulp_modules/features/antagonists/bloodsuckers/code/clans/clan_flavortext.dm

```diff
@@ -17,31 +17,32 @@
 	. = ..()
 	var/area/current_area = get_area(bloodsuckerdatum.owner.current)
 	if(istype(current_area, /area/station/service/chapel))
-		to_chat(bloodsuckerdatum.owner.current, span_warning("You don't belong in holy areas! The Faith burns you to a crisp!"))
+		to_chat(bloodsuckerdatum.owner.current, span_warning("You don't belong in holy areas! The faith burns you to a crisp!"))
```

Comment:
> This one is intended i think
> 
> https://whitewolf.fandom.com/wiki/True_Faith

From the understanding I've developed of it after a quick glance at the wiki page, "True Faith" seems more like a specific ability or trait that would be associated with a person rather than a place. I can still change it back if you want me to though.